parlai/crowdsourcing/tasks/dialcrowd/README.md

```diff
@@ -0,0 +1,116 @@
+# DialCrowd
```

Comment:
Would be very useful to include some information on what DialCrowd does, high-level aims, etc. here

Reply:
Hi! Thank you for the response; we do have a paper describing a previous version of DialCrowd available (https://aclanthology.org/W18-5028/), but a paper describing this version is yet to be available.

For the configuration page, this is to set up config.json, which populates the annotation page. We have sections which walk a requester through all the steps necessary to set up a HIT with this UI.
![ss11](https://user-images.githubusercontent.com/8203518/156258103-d6bd090f-ec0f-4632-8568-55031e8ac912.PNG)

For the results page, this is to view the results afterward. It does some basic analysis, including checking if the golden data responses match, if a worker took an abnormally long or short time on the HIT, etc.
![ss12](https://user-images.githubusercontent.com/8203518/156258256-a6a6d370-cbc3-4d5a-8c5d-2250aba640b4.PNG)

We also have a site, https://cmu-dialcrowd.herokuapp.com/, of which the intent classification tab has approximately the same functionalities as the template we have integrated into the ParlAI system.

Do let me know if there is an easier way to do any of these pages (and what other information you would want)!

Reply:
oh yeah thanks for this detail! Hmm would you be able to add this info to the README itself so that people can see it first thing when they view the README? It'd also be useful to add in your info in the PR description, "DialCrowd is a dialogue crowdsourcing toolkit that helps requesters write clear HITs, view and analyze results, and obtain higher-quality data. This integration allows for the requester interface, worker interface, and analysis interface to be integrated into ParlAI so requesters can have access to ParlAI's tools with DialCrowd's tools."